projects/dashboards/src/widgets/widget-renderer.ts

```diff
@@ -45,7 +45,7 @@ export abstract class WidgetRenderer<TModel extends object, TData = unknown> imp
     this.fetchAndRunChangeDetection();
   }
 
-  private onDashboardRefresh(): void {
+  protected onDashboardRefresh(): void {
```

Comment:
This should remain private - will suggest an alternative when I see how it's used

Reply:
Fixed